src/app/modals/createwallet/createwallet.component.html

```diff
@@ -64,7 +64,7 @@
           </button>
         </div>
         <div class="col-sm-6 col-xs-12 text-left">
-          <button class="control" type="submit" [disabled]="!nameForm.form.valid">
+          <button class="control" type="submit" [disabled]="!nameForm.form.valid" [appFocusTimeout]="nameForm.form.valid">
```

Comment:
This removes focus from the name field.. Maybe change it to if the enter key is hit in the input, next step